server/BUILD

```diff
@@ -160,7 +160,35 @@ linux_packages(
         "//server:services/cassandra/logback.xml": "services/cassandra/logback.xml",
         "//server:services/grakn/grakn-core-ascii.txt": "services/grakn/grakn-core-ascii.txt"
     },
-    modes = {
+    permissions = {
```

Comment:
same issue, remove `./` in `permissions`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c